nikola/__init__.py

```diff
@@ -26,11 +26,14 @@
 
 """Nikola -- a modular, fast, simple, static website generator."""
 
-from __future__ import absolute_import
 import os
+import sys
 
 __version__ = '7.8.8'
```

Comment:
We generally change version numbers right before release. I’ll make it `8.0.0.dev0`.